_posts/2022-11-01-segmentation.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Segmentation Part I"
```

Comment:
```suggestion
title: "Segmentation software usability and performance: Part I"
```

Segmentation Part I is not very descriptive. Please update my title suggestion if you'd like!

Reply:
Agreed! I like this title and "usability" was totally the word I was looking for but could never place!